dgl_sparse/include/sparse/spspmm.h

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
It is probably a little bit overkill to create one header per method. In the future, the customers need to include multiple header files to do one thing.

Maybe merge them all to mm.h?